nlp/2021_06_18_data_augmentation/README.md

```diff
@@ -0,0 +1,10 @@
+# Transformer-based Data Augmentation
+
+Ever struggled with having a limited non-English NLP dataset for a project? 🤯 Fear not, data augmentation to the rescue ⛑
```

Comment:
the '⛑' doesn't display very properly here, nor in the readme (https://github.com/ml6team/quick-tips/blob/feature/nlp_data_aug/nlp/2021_06_18_data_augmentation/totw_nlp_dat_aug.ipynb).

Always a good idea to check on all platforms to see how it renders (readme, github jupyter, colab jupyter, etc.)

-